dask/compose.py

```diff
@@ -0,0 +1,66 @@
+from uuid import uuid4
+import operator
+from functools import partial, wraps
+
+from toolz import merge
+
+from .optimize import cull
+from .context import _globals
+from . import threaded
+
+
+def get_uuid_dasks(v):
+    if isinstance(v, Value):
+        return v._uuid, v._dasks
+    else:
+        return v, []
+
+
+def applyfunc(func, *args, **kwargs):
+    func = partial(func, **kwargs)
```

Comment:
```python
if kwargs:
    func = ...
```

This greatly helps inspectability of the graph.

When kwargs exists then we might even want to make a function like

```python
def apply(func, args, kwargs):
    return func(*args, **kwargs)
```

and use this within the graph

```python
(apply, func, args, kwargs)
```
 to expose both the function name and the kwargs